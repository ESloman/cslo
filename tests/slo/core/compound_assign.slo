var x = 10;
x += 5;
println(x); // should print 15
x -= 2;
println(x); // should print 13
x *= 3;
println(x);  // should print 39
x /= 3;
println(x);  # should print 13
