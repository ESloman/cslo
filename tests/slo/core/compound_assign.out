15
13
39
13
